docs/data/material/components/modal/SpringModal.tsx

```diff
@@ -64,9 +64,11 @@ export default function SpringModal() {
         open={open}
         onClose={handleClose}
         closeAfterTransition
-        BackdropComponent={Backdrop}
-        BackdropProps={{
-          timeout: 500,
+        slots={{ backdrop: Backdrop }}
+        slotProps={{
+          backdrop: {
+            TransitionComponent: Fade,
+          },
```

Comment:
For a consistent animation, use the Fade component for both the modal body and modal backdrop.